components/Home/InstallTabs/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { useEffect, useState } from 'react';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import WindowsPanel from './WindowsPanel';
+import MacOSPanel from './MacOSPanel';
+import LinuxPanel from './LinuxPanel';
+import styles from './index.module.scss';
+import { detectOS } from '../../../util/detectOS';
+import type { UserOS } from '../../../types';
+import type { FC, ReactNode } from 'react';
+
+const getOSPanel: FC<UserOS> = userOS => {
+  switch (userOS) {
+    case 'MAC':
+      return (
+        <>
+          <TabPanel
+            className={styles.reactTabsTabPanel}
```

Comment:
Small suggestion - perhaps create a styles object:

```tsx
const tabPanelStyles = {
  className: styles.reactTabsTabPanel,
  selectedClassName: styles.reactTabsTabPanelSelected
};
  ```
  
  And spread it into each tab panel: `<TabPanel {...tabPanelStyles}>`

Reply:
Done thanks for the idea!